internal/pkg/aws/route53/route53.go

```diff
@@ -79,3 +165,28 @@ func matchesDomain(domain string) filterZoneFunc {
 		return domain == aws.StringValue(z.Name) || domain+"." == aws.StringValue(z.Name)
 	}
 }
+
+func cleanNSRecord(record string) string {
+	if !strings.HasSuffix(record, ".") {
+		return record
+	}
+	return record[:len(record)-1]
+}
+
+func eqStringSlices(first, second []string) bool {
```

Comment:
This function seems useful beyond this application-- I wonder if we should move it to somewhere more general?